news/2018-03-06-new-featured-artist-trial-and-error.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: "New Featured Artist: Trial & Error"
+date: 2018-03-06 11:00:00 +0000
+---
+
+The search for new musical talent never ends, and we're proud to announce the next addition to osu!'s Featured Artist lineup - **Trial & Error**!
```

Comment:
musical talents

Reply:
talent singular here implies the general search for talent overall instead of plural talents being "multiple", if that makes sense

so the sentence implies that we're after musical skill wherever it is rather than looking for it in specific domains

i think it's an english quirk